core/src/utils/test/playwright/docs/usage-instructions.md

```diff
@@ -59,7 +59,13 @@ The examples provided in the [Running Tests](#running-tests) section also apply
 Note that since you are generating the reference branch ground truth screenshots, you must be on the reference branch locally. Don't forget to pull the latest reference branch changes and then re-build using `npm run build`.
 
 ```shell
-npm run test.e2e -- --update-snapshots
+npm run test.e2e.update-snapshots
```

Comment:
Can we mention the `update-snapshots` flag from https://playwright.dev/docs/test-snapshots so there's a clear connection between our custom script `npm run test.e2e.update-snapshots` and the underlying Playwright functionality? similar to what we do above with `npm run test.e2e` and `npx playwright test`